packages/agent/src/display.ts

```diff
@@ -434,22 +437,35 @@ function safeCalculateJwkThumbprint(jwk: JwkJson): string | undefined {
   }
 }
 export function getAttributesAndMetadataForMdocPayload(namespaces: MdocNameSpaces, mdocInstance: Mdoc) {
+  const topLevelMetadataFields = ['issue_date', 'expiry_date']
+
   const attributes: CredentialForDisplay['attributes'] = Object.fromEntries(
-    Object.values(namespaces).flatMap((v) =>
-      Object.entries(v).map(([key, value]) => [key, recursivelyMapAttribues(value)])
-    )
+    Object.values(namespaces).flatMap((v) => {
```

Comment:
I noticed a mismatch in the count when sharing from an mdoc because of some metadata properties (issue and expiry date) are visible. 

I fixed it like this, but this might cause issues for real props named expiry date?